docker/images/proxysql/tarball-compliant/entrypoint/entrypoint.bash

```
@@ -0,0 +1,51 @@
+echo "==> Cleaning staging directories"
+rm -rf "/opt/proxysql/binaries/${TARBALL_FILE}" "/opt/proxysql/pkgroot" || true
+
+echo "==> Building ProxySQL"
+cd /opt/proxysql
+git config --system --add safe.directory '/opt/proxysql'
+
+# Determine targets
+deps_target="build_deps_clickhouse"
+build_target="clickhouse"
+
+${MAKE} ${MAKEOPT} ${deps_target}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)